README.md

```diff
@@ -1,86 +1,99 @@
+#### Tool Initialization
+The tool should be initialized with the server details of the environment it is run against.
+1. Create a new folder and open a terminal inside it.
+2. Run the following command to create the configuration files needed to initialize the tool.
     ```
-    alias iamctl="<IDENTITY-CLI-PATH>/bin/iamctl" 
+    iamctl setupCLI
     ```
-  
-* mac:
- 
-  ```
-   alias iamctl="<IDENTITY-CLI-PATH>/bin/iamctl" 
-  ```
+3. A new folder named ```configs``` will be created with a ```env``` folder inside it, having 2 configuration files: ```serverConfig.json``` and ```toolConfig.json```.
+> **Note:** If you have multiple environments, get a copy of the ```env``` folder and rename it according to the environments you have. For example, if you have 2 environments: dev and prod, have 2 separate config folders as ```dev``` and ```prod```. 
+4. Open the ```serverConfig.json``` file and provide the IS server details and client ID/secret of the app you created earlier.
 
-* windows
+Example configurations:
 
     ```
-    doskey iamctl=<IDENTITY-CLI-PATH>/bin/iamctl.exe $*
+    "SERVER_URL" : "https://localhost:9443",
+    "CLIENT-ID" : "********",
+    "CLIENT-SECRET" : "********",
+    "TENANT-DOMAIN" : "carbon.super"
     ```
- 
- 
-5. Now you can run IAM-ctl using your keyword.
-```
-iamctl -h
+
+#### Export
+Run the following command to export all supported resource configurations from the target environment to the current directory.
 ```
-It gives following details.
+iamctl exportAll -c ./configs/env
 ```
-:~$ iamctl -h
-Service Provider configuration
-
-Usage:
-  IAM-CTL [flags]
-  IAM-CTL [command]
-
-Available Commands:
-  application         Create a service provider
-  help                Help about any command
-  init                you can set your sample SP
-  serverConfiguration you can set your server domain
+A set of new folders will be created, named after each resource type, with exported yaml files for each available resource in the IS server.
 
-Flags:
-  -h, --help   help for IAM-CTL
-
-Use "IAM-CTL [command] --help" for more information about a command.
+#### Import
+Run the following command to import all supported resource configurations from the current directory to the target environment.
+```
+iamctl importAll -c ./configs/env
 ```
-6. First you need to configure service provider to get access from identity server. For that this [link](https://docs.wso2.com/display/IS570/Configuring+OAuth2-OpenID+Connect+Single-Sign-On) will help to you..
-7. Then use  client_key, client_secret of created service provider to do the authorization relevant to server domain. It should be completed as follows.
+All resources available inside each resource type folder in the current directory will be imported to the IS server.
+
+## Interactive Mode
+The interactive mode can be used to handle application configurations in an interactive manner. This can be used to add, list, export and import applications in the target environment.
```

Comment:
```suggestion
The interactive mode can be used to handle application configurations in an interactive manner. This can be used to add, list, export, and import applications in the target environment.
```